client/package.json

```diff
@@ -27,5 +29,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@testing-library/react": "^9.3.0",
+    "jest-dom": "^4.0.0",
+    "react-testing-library": "^8.0.1"
```

Comment:
`@testing-library/react` is the only one you really need here.
Adding in `jest-dom` gives you some extra features, so that's helpful

`react-testing-library` is just the older version of `@testing-library/react`, so it's un-needed here '' '